src/shared-components/accordion/style.ts

```diff
@@ -16,11 +15,11 @@ export const ExpansionWrapper = styled.div<{ contentHeight: string }>`
   transition: max-height ${ANIMATION.defaultTiming} ease-in-out;
 `;
 
-const getBorderStyle = (theme: ThemeType, isOpen: boolean) => css`
+const getBorderStyle = (theme: ThemeType, isOpen: boolean) => `
   border: 1px solid ${theme.COLORS.border};
 
   ${ExpansionWrapper} {
-    ${isOpen && `border-top: 1px solid ${theme.COLORS.border}`};
+    ${isOpen ? `border-top: 1px solid ${theme.COLORS.border};` : ''}
```

Comment:
Since I removed the serialized style, I inadvertently broke some sanitizing of the return object `css` was providing us. Now we explicitly add this semi-colon.